### PR TITLE
Fixed Chages To Design

### PR DESCRIPTION
**Pull request by sj29-innovate**
I have made changes to the design .Please provide your feedback.

### CODE REVIEW

**Comment by mariobehling** [comment]
@sj29-innovate Thanks. Please provide screenshots and a link to a setup on your repo.

Workflow on FOSSASIA projects is, that issues should be opened first, where other devs could give feedback, if needed. It also helps to keep the development more transparent.

Every pull request should be matched with an issue then. Please help to make it easier for reviewers by following this flow.

Thank you!

**Comment by sj29-innovate** [comment]
As per your feedback i have reduced inline styles and styling through js and also i have linked the PR with issue #4 
new1
new2
new3

**Comment by mariobehling** [comment]
Thank you. Please resolve merge conflicts and link to a site you have set up for tests.

File: yacysearch/index.html [hunk 1]
```diff
@@ -155,7 +155,7 @@
     <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main" id="main">
       <form class="search small" id="newform" name="searchform" action="#" method="get" accept-charset="UTF-8">
         <div class="input-group">
-          <a href="javascript:{}" id="newbtn1"><img id="newbtn1img" src="../images/search.png" alt="Search"></a>
+      <!--    <a href="javascript:{}" id="newbtn1"><img id="newbtn1img" src="../images/search.png" alt="Search"></a>-->
```
Comment (hemantjadon) [inline yacysearch/index.html]: Why using the comment, If we need to remove this line remove this completely.

**Comment by sj29-innovate** [comment]
@hemantjadon  Can you please review this PR

**Comment by hemantjadon** [comment]
@sj29-innovate i think the PR is okay, can you just squash the commits so that we can merge this PR ??

**Comment by sj29-innovate** [comment]
@hemantjadon My tests are failing and Travis CI shows errors saying remove white spaces and due to tabs.Do i need to remove all that because there are a lot of them and it was in the code that i had cloned initially.

**Comment by mariobehling** [comment]
Future project has move to new repo in the FOSSASIA project. Closing this PR therefore. Thanks!

**Pull request closed without merging**